src/main/webapp/includes/download-box-spark.jspf

```diff
@@ -74,6 +79,46 @@
                         <span class="ignite_download_item_date"><%= buildDate %></span>
                         <span class="ignite_download_item_size"><%= formatter.format((double)binExe.length()/1024.0/1024.0) %> MB</span>
                     </div>
+                    <div class="ignite_download_item_even">
```

Comment:
I think you want to alternate `ignite_download_item_odd` / `ignite_download_item_even` here.

Reply:
Oh yeah, I didn't notice that, I fixed it.